views/login.html

```diff
@@ -1,6 +1,30 @@
 <!-- Authentication Form -->
 <div id="authentication">
-  <!-- Sign Up Forms -->
+  <!-- Sign In Form -->
```

Comment:
*Some questions I have* 
1. Why does Github show in _green_ that you added _(+)_ 25 changes
    -  Or does Github just show that 25 lines have been changed?
    - NOT the actual changes you made to the code itself?